# HG changeset patch
# User Rik <rik@octave.org>
# Date 1471371829 25200
#      Tue Aug 16 11:23:49 2016 -0700
# Node ID a1840b139c377654c15edebfe38080f45e62ba49
# Parent  7cd75d61bbae9db68c34b9adabad12d74a050b3f
dump_demos.m: Remove dummy assert function.

* imshow.m: Add missing quote to title().

* dump_demos.m: Remove dummy assert function.

diff --git a/scripts/image/imshow.m b/scripts/image/imshow.m
--- a/scripts/image/imshow.m
+++ b/scripts/image/imshow.m
@@ -241,17 +241,17 @@ endfunction
 %! clf;
 %! imshow (rand (100, 100));
 %! title ({"imshow with random 100x100 matrix", "black and white"});
 
 %!demo
 %! clf;
 %! imshow (rand (100, 100));
 %! colormap (jet (64));
-%! title ({"imshow with random 100x100 matrix, "colormap() makes color image"});
+%! title ({"imshow with random 100x100 matrix", "colormap() makes color image"});
 
 %!demo
 %! clf;
 %! imshow (rand (100, 100, 3));
 %! title ({"imshow with random 100x100x3 matrix", "RGB color"});
 
 %!demo
 %! clf;
diff --git a/scripts/testfun/private/dump_demos.m b/scripts/testfun/private/dump_demos.m
--- a/scripts/testfun/private/dump_demos.m
+++ b/scripts/testfun/private/dump_demos.m
@@ -284,18 +284,12 @@ function dump_helper_fcns (fid)
 "    h = htmp;                                                                "
 "  end                                                                        "
 "                                                                             "
 "end                                                                          "
 ]);  # End of rgbplot dump
 
   fprintf (fid, "\n");
 
-  ## Add dummy assert until we've removed all assert from demos
-  fdisp (fid, [
-"function assert (varargin)                                                   "
-"end                                                                          "
-]);  # End of dummy assert
-
   fprintf (fid, "\n%s\n", repmat ("%", [1, 60]));
 
 endfunction
 
